src/index.ts

```diff
@@ -50,6 +50,7 @@ export { default as Tab } from './components/Tab';
 export { default as Tabs } from './components/Tabs';
 export { default as Tag } from './components/Tag';
 export { default as Text } from './components/Text';
+export { default as TextField } from './components/TextField';
```

Comment:
Let's add `Label`, `TextInput`, and `FieldNote` to this `index.ts` as well once those are moved into components

Reply:
ADded